tests/figs/deps.txt

```diff
@@ -1,3 +1,3 @@
 - vdiffr-svg-engine: 1.0
 - vdiffr: 0.3.3
-- freetypeharfbuzz: 0.2.6
+- freetypeharfbuzz: 0.2.5
```

Comment:
@thomasp85 Not entirely sure why this change happened. It doesn't seem to affect any of the reference images. If you think it matters, please let me know what I need to do to change this back. I updated to the latest vdiffr but that didn't seem to make a difference.

Reply:
I think it is because you have an older version of freetypeharfbuzz installed and it records the version last used to validate the tests